site/content/en/docs/enterprise/subscription-managment.md

```diff
@@ -64,8 +72,113 @@ All limits of the **Free** plan will be removed.
 > So, if you have three annotators working, you'll need to pay
 > for 4 seats (3 annotators + 1 organization owner).
 
-To add **VAT** or **Organization** details to the invoice, after subscribing to the Team plan,
-go to **Manage Team Plan** > **Manage** > **Billing information**.
+### Can a paid invoice be modified?
+
+Once an invoice has been paid, it is not possible to modify it. This restriction is due to the
+limitations of the payment processing platform used, which in the case of CVAT.ai, is [Stripe](https://stripe.com/).
+
+Stripe's policy dictates that revisions to an invoice can only be made before payment.
+For more comprehensive information on this policy, please refer to Stripe's official documentation
+on [invoice edits](https://docs.stripe.com/invoicing/invoice-edits#:~:text=Stripe%20lets%20you%20revise%20a,Edit%20the%20invoice%20description.)
+at their website.
+
+### How to add VAT/tax and other information to the CVAT.ai invoice before the first payment?
+
+To ensure VAT (tax) information and other relevant details are included on your CVAT.ai invoices,
+it's important to add this information before making the first payment.
+
+This process involves using the Stripe subscription management panel, where you'll need to
+enter your VAT and other organizational details.
+
+Here’s how you can do it:
+
+1. Sign up for a CVAT.ai account and log in.
+2. Create an [organization](/docs/manual/advanced/organization/) and switch to an **Organization** account.
```

Comment:
Basically, the same instructions should be valid for the Solo plan. I have no idea why you describe it for the team plan specifically.